money_market_account.rb

```diff
@@ -0,0 +1,53 @@
+module  Bank
+
+  class MoneyMarketAccount < Account
+    attr_reader :balance, :owner, :account_id, :owner_id, :good_standing, :max_trans
+
+    MIN_BALANCE = 1000000
+
+    def initialize(id, initial_balance, open_date)
+      super
+      @max_trans = 0
+      @good_standing = true
+    end
+
+    def withdraw(amount)
+      new_balance = @balance - amount
+      if @max_trans >= 6 || @good_standing == false
+        puts "No more transactions available."
+      elsif amount < 0
+        puts "You can't withdraw a negative amount."
+      elsif new_balance < self.class::MIN_BALANCE
+        @balance = @balance - amount - 10000
+        @good_standing =  false
+        puts "Next time you need to deposit enough money to bring your account up to #{self.class::MIN_BALANCE}"
+        @max_trans += 1
+      elsif new_balance < 0 
+        puts "That will make your balance negative. Please try a smaller amount."
+        return @balance
+      else
+        @max_trans += 1
+        @balance = new_balance
+        return @balance
+      end
+    end
+
+    def deposit(amount)
+      if @balance < 1000000
+        super(amount)
```

Comment:
Since both of these conditionals are doing `super(amount)` I think you could abstract this out of your conditional and then simplify to do only the `@max_trans += 1` operation